src/Client/components/NavBar.tsx

```
@@ -0,0 +1,121 @@
+                                            My Profile
+                                        </button>
+                                    </DropdownItem>
+                                    <DropdownItem key="settings">
+                                        <button
+                                            onClick={() =>
+                                                router.push("/mysettings")
+                                            }
+                                            className={`flex items-center gap-2`}
+                                            style={{
+                                                color: "inherit",
+                                                textDecoration: "none",  }}  >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)